src/web/styles.scss

```diff
@@ -3,46 +3,12 @@
 @import '~bootstrap/dist/css/bootstrap.css';
 @import '~@fortawesome/fontawesome-free/css/all.css';
 @import '~handsontable/dist/handsontable.full.css';
-@import '~@angular/material/prebuilt-themes/deeppurple-amber.css';
 
 body {
   font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
   font-size: 15px;
 }
 
-// make snackbar over the modal
```

Comment:
I guess we are still making sure that the toast is over the modal?

Reply:
https://github.com/TEAMMATES/teammates/pull/10206/files#diff-e09b073016a369e781a6cbad2f55ac23R7-R8